include/macros.h

```diff
@@ -50,9 +50,16 @@
 
 #define CURRENT_DAY (((void)0, gSaveContext.day) % 5)
 
-#define CLOCK_TIME(hr, min) ((s32)(((hr) * 60 + (min)) * 0x10000 / (24 * 60)))
+#define TIME_TO_MINUTES(time) (s32)((time) * ((24 * 60) / 0x10000))
+#define CLOCK_TIME(hr, min) (s32)(((hr) * 60 + (min)) * 0x10000 / (24 * 60))
```

Comment:
I think the only "reversed" one I have seen is 0x3FFC (there may be more that I'm not remembering right now), and that seems to be related to z_schedule.
Since z_demo.c doesn't use that macro, I didn't add it, because it felt out of scope for the PR